### ISSUE TITLE
Choards

### ISSUE THREAD

**Issue by sofakid** [sofakid/Skoarcery#38]
Am, C#, Bsus2, D7m7, aaaallll the ways.

**Comment by lukeorland** [comment]
Does this mean that choards don't work yet? I tried to use them, but only got single notes, with the default instrument.

**Comment by sofakid** [comment]
Yes, at the moment, the choards are recognized, but not really implemented.

**Comment by sofakid** [comment]
More work is needed here: https://github.com/sofakid/Skoarcery/blob/master/SuperCollider/Skoar/pitchy.sc#L205

**Comment by sofakid** [comment]
I did some work on Choards, they mostly work now.. I still need to implement different voicings, (Cm~~ would be Cm shifted left/down twice)

The grammar accepts shifted voicings, but they won't actually happen. I don't want this closed until that works too.
